src/PKSim.Core/Services/PopulationAnalysisFlatTableCreator.cs

```diff
@@ -202,7 +201,35 @@ private static void storeReferenceGroupingItemOn(PopulationAnalysisDerivedField
 
       private string getReferenceExpression(string referenceSimulationName, string referenceLabel, string elseCase)
       {
-         return $"iif([{_simulationNameColumnName}]='{referenceSimulationName}', '{referenceLabel}', {elseCase})";
+         return $"iif([{_simulationNameColumnName}]='{escapeField(referenceSimulationName)}', '{referenceLabel}', {elseCase})";
+      }
+
+      private string escapeField(string fieldToEscape)
```

Comment:
ok somehow we need to escape special chars (and more). So I got this methods from chatGPT and it works :)

Reply:
nice :)